.woodpecker.yml

```
@@ -242,7 +242,6 @@ steps:
       platforms: linux/amd64,linux/arm64
       build_args:
         - RUST_RELEASE_MODE=release
```

shouldn't this be setting the git tag now?
```suggestion
        - RUST_RELEASE_MODE=release
      tag: ${CI_COMMIT_TAG}
```

---

Yep looks correct. https://woodpecker-ci.org/plugins/Docker%20Buildx